server/plugins/ansible/impl/src/main/java/com/walmartlabs/concord/server/plugins/ansible/ProcessAnsibleResource.java

```diff
@@ -547,4 +558,16 @@ public enum TaskType {
         SETUP,
         HANDLER
     }
+    
+    public enum SortField {
```

Comment:
Maybe make enum values uppercase for consistency's sake and convert to lower later?